modules/nf-core/mgikit/demultiplex/main.nf

```diff
@@ -0,0 +1,88 @@
+process MGIKIT_DEMULTIPLEX {
+    tag {"$run_id"}
+    label 'process_high'
+
+    conda "${moduleDir}/environment.yml"
+
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/mgikit:0.1.6--h4c94732_0' :
+        'biocontainers/mgikit:0.1.6--h4c94732_0' }"
+
+    input:
+    tuple val(meta), path(samplesheet), path(run_dir)
```

Comment:
```suggestion
    tuple val(meta), path(samplesheet)
    path run_dir
```

Reply:
This is to follow the same input in other demultiplexers in the demultiplex pipeline like bcl2fastq

Reply:
bcl2fastq module is [here](https://github.com/nf-core/modules/blob/master/modules/nf-core/bcl2fastq/main.nf). I do prefer your suggestion.